meta-lmp-base/classes/lmp.bbclass

```diff
@@ -3,7 +3,9 @@ FIO_PUSH_CMD ?= "fiopush"
 FIO_CHECK_CMD ?= "fiocheck"
 SOTA_TUF_ROOT_DIR ?= "usr/lib/sota/tuf"
 
-IMAGE_FSTYPES += "${@bb.utils.contains('EFI_PROVIDER', 'systemd-boot', 'ota-esp', ' ', d)}"
+IMAGE_FSTYPES_OTA_ESP ?= ""
+IMAGE_FSTYPES_OTA_ESP:sota = "${@bb.utils.contains('EFI_PROVIDER', 'systemd-boot', 'ota-esp', '', d)}"
```

Comment:
I would just make this variable a bit more generic, like LMP_OTA or simply OTA instead of OTA_ESP.

Reply:
`IMAGE_FSTYPES_OTA_ESP` renamed to `IMAGE_FSTYPES_OTA`.